tests/containers/covid19/Covid19Container-test.jsx

```diff
@@ -26,7 +26,15 @@ jest.mock('containers/covid19/WithDefCodes', () => ({
 
 // Mock the custom hook useQueryParams
 jest.mock('helpers/queryParams', () => ({
-    useQueryParams: jest.fn()
+    useQueryParams: jest.fn(),
+    getQueryParamString: jest.fn().mockImplementation((obj) => {
```

Comment:
Could we use the real function or just mock the return value in the tests below? It seems counter-intuitive to recreate this functionality in our test file. In other words, when we "expect" a certain search string below, it is testing this implementation instead what actually gets run by the app.

Reply:
Its a built in function on the browser and doesnt exist in the test env.